src/styles/index.scss

```diff
@@ -1,3 +1,30 @@
-body {
-  margin: 0;
+$column-count: 1;
+
+.container {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat($column-count, $card-width);
```

Comment:
The variable `$card-width` is used here, but it is not defined in the provided code. Make sure to define `$card-width` in your SCSS file or import it from another file where it is defined.